### PR TITLE
Finance ctte role update

### PR DESCRIPTION
**Pull request by jdswinbank**
Trivial changes for consistency

### CODE REVIEW

File: roles.json [hunk 1]
```diff
@@ -131,9 +131,9 @@
         "responsibilities": {
             "description": "Plan, oversee, and disburse the Astropy project finances, including:",
             "details": [
-                "Determine the process for paying people from Astropy Project-level funding.",
+                "Determine and manage the process for paying people from Astropy project-level funding.",
```
Comment (pllim) [inline roles.json:134]: When you change "project" to lowercase, the meaning is a bit different? I thought "Astropy Project" is a proper noun?
```suggestion
                "Determine and manage the process for paying people from Astropy's project-level funding.",
```

Comment (jdswinbank) [reply roles.json:134]: I don't disagree with you, but I was aiming for consistency with the rest of the page — it starts “The Astropy project...”!

I like your wording though; it neatly sidesteps the issue (in this text, at least). 👍

**Review by eteq — APPROVED** [review]

**Comment by eteq** [comment]
(the CI failure is unrelated)

**Comment by pllim** [comment]
Re: CI -- `ERROR: Home URL for astroML - https://astroml.github.com/ - did not work` -- @bsipocz ?

**Comment by pllim** [comment]
p.s. Would be funny though if CI actually objected to this PR change.

**Pull request merged**